CONTRIBUTING.md

```diff
@@ -26,6 +26,8 @@ You will need to submit a CLA before we can accept your contribution. You only h
 
 > For cross compilation on MacOS follow [these](https://github.com/temporalio/sdk-node/blob/main/docs/building.md) instructions.
 
+To update your environment, run `git submodule update` to update to the latest version of the Core SDK, followed by `npm ci` to recompile.
```

Comment:
Better yet, `npm run build` after updating the submodule.
No need to reinstall the npm modules.